### PR TITLE
Read from stdin if no file is passed

### LINKED ISSUE #3: accept stdin input
not just a filename

### PR DESCRIPTION
**Pull request by remram44**
Fixes #3

Liked the talk last night :smile:

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -32,59 +32,69 @@ struct Args {
...
-    let args: Args = Docopt::new(USAGE)
-        .and_then(|d| d.decode())
-        .unwrap_or_else(|e| e.exit());
-
-    // just for now.
-    let filename = args.arg_file.unwrap();
-    let file = File::open(&filename).ok().expect("I couldn't open that file, sorry :(");
-    let reader = BufReader::new(file);
+struct CountResults {
+    bytes: usize,
```
Comment (steveklabnik) [inline src/main.rs]: `usize` as a type is only appropriate when dealing with something related to memory size, like indexing. I would prefer that these are `u64`.

**Comment by steveklabnik** [comment]
This looks great! Just one nit. Thanks so much.

(and glad you liked the talk :) )

File: src/main.rs (hunk 1 above)
Comment (remram44) [inline src/main.rs]: Iterator::count() and String::len() return usize, so anything else will require casts.

Comment (steveklabnik) [inline src/main.rs]: Yeah, that makes sense. Hmmmmmmmmmm

**Comment by steveklabnik** [comment]
@remram44 this is now out of date, if you'd like to rebase, i will merge :)

**Comment by remram44** [comment]
Now with `u64`. Still doesn't handle multiple arguments (output format would need to be different).

**Comment by steveklabnik** [comment]
Looks great! Thanks!

**Pull request merged**